evaluators/text/RougeEvaluator/__init__.py

```diff
@@ -0,0 +1,27 @@
+from typing import Sequence
+
+from rouge import Rouge
+
+from jina.executors.evaluators.text import BaseTextEvaluator
+
+class RougeEvaluator(BaseTextEvaluator):
+    """
+    :class:`RougeEvaluator` Evaluate Rouge score between acutal and ground truth.
+    """
+
+    def __init__(self, metrics: str='rouge-1', stats: str='r', *args, **kwargs):
+        """metrics: can be rouge-1, rouge-2 or rouge-l
```

Comment:
Typo I guess, do u mean default rouge 1? Can u pass a reference to see the differences?

Reply:
This is a requirement from the library

Reply:
Ok missread rouge-1 and rouge-l. Can you put in comments a link to the lib documentation?